gadget-container/gadgettracermanager/main.go

```diff
@@ -120,6 +122,11 @@ func init() {
 }
 
 func main() {
+	args,err:=entrypoint.Init()
+	if err != nil {
+		log.Fatalf("entrypoint.Init() failed: %v", err)
+	}
+	os.Args=args
```

Comment:
I'd change this a little bit:
1. Make this function take the hook-mode as argument. 
2. Return the mutated hookmode value needed to use for the gadget tracer manager 
3. Move the logic to handle fallback pod informer directly here, that function is doing nothing for it

Then this function shouldn't access the configuration directly, but instead should only rely on parameters passed. 

Also, the invocation of this should be moved under the `if serve` conditional below, at that point the configuration is available, so getting the value of the hook point is possible. 

After these changes, the manifests should be updated to pass the `-serve` and `-controller` arguments to gadgettracermanager.

Please let me know if you have any changes. 

Additionally, please be sure you run `make lint`.

Reply:
Hey @mauriciovasquezbernal, sorry for the delay! I've been caught up with some things lately but will get to this very soon.

Reply:
Hey @mauriciovasquezbernal, sorry for the delay! I've been caught up with some things lately but will get to this very soon.

Reply:
Hi @mauriciovasquezbernal,

I’ve made the requested changes in my PR. Here’s a summary of the updates:

 -  The `Init` function now takes `hookMode` as an argument.
 -  It returns the mutated `hookMode` value as needed.
 -  The logic for handling the fallback pod informer has been moved to the `main` function.
 -  I’ve linted the code to ensure proper formatting.
 -  The manifests have been updated to include the `-serve` and `-controller` arguments.

Please let me know if anything else is needed.